src/jsonValidation/appIntroValidator.ts

```diff
@@ -0,0 +1,34 @@
+import { isArray, isObjectLike, isString } from 'lodash';
+
+import { Initializer, Initializers } from '../helpers/initializationHelper';
```

Comment:
do you mind putting initializationHelper in index.js for helpers to be able to import from "../helpers" at all places?

Reply:
![Screenshot 2021-11-25 at 11 19 17](https://user-images.githubusercontent.com/59824597/143423444-e15f55ed-3dd4-411c-980b-f9114ede459e.png)

no, as that creates require cycles.